### ISSUE TITLE
request: inlet for [receive] to set receive symbol

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#603]
this would free me from the dependency of using [r.mmb]

**Comment by porres** [comment]
this is particularly useful to create GUI abstractions that behave like other GUIs, where you can give it a 'receive' name via an input message from the inlet

**Comment by jaffasplaffa** [comment]
Just asked about this today. I hadn't seen that it was already requested, so I'll post here instead.

Would be great to have this :) 

Maybe is can be made so it is also possible to use a "set name" and not only "symbol name", basically the same way as the tabread uses a "set" message to set a name dynamically.

Cheers!

**Comment by Spacechild1** [comment]
> Maybe is can be made so it is also possible to use a "set name" and not only "symbol name",

why add two ways to do the same thing?

> basically the same way as the tabread uses a "set" message to set a name dynamically.

IMO, `[receive]` should match the semantics of `[send]`, and not that of unrelated objecs like `[tabread]`

**Comment by jaffasplaffa** [comment]
I think "set" makes more sense than "symbol", as it actually describes what it does. 

And since the send object already uses "symbol", changing it to "set" would break backwards compatibility?

So that's why I suggested to add it, so it won't break anything.

**Comment by porres** [comment]
> But yeah, using "symbol" will also work, I guess, if thats the norm for that line of objects :)

yeah, cause then they're consistent between each other, it'd be weird that [r] has "set" but [s] doesn't - and it'd be really hard to add a set method to the second inlet of [s]

**Comment by jaffasplaffa** [comment]
Yes, they need to be identical, I agree and also be backwards compatible.

But set just makes more sense than symbol, imo.

so symbol is probably the way to go, then it won't make too much of a fuzz :)

**Comment by Jonathhhan** [comment]
great idea. i think the symbol in the message is just needed to format the message, you can also send a symbol directly into the second inlet from [send]. the same should be true for receive?
